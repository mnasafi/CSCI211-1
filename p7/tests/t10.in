echo t10: Insert days of week into tree, then call print
insert Friday
insert Monday
insert Saturday
insert Sunday
insert Wednesday
insert Thursday
insert Tuesday
print
