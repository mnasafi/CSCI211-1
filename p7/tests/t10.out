t10: Insert days of week into tree, then call print
{Friday, Monday, Saturday, Sunday, Thursday, Tuesday, Wednesday}
